src/SitemapWriterXml.ts

```diff
@@ -47,7 +47,7 @@ export default class SitemapWriterXml extends SitemapWriterFile {
                 `<lastmod>${formatLastModified(entry.lastModified)}</lastmod>`,
               entry.changeFrequency &&
                 `<changefreq>${entry.changeFrequency}</changefreq>`,
-              entry.priority && `<changefreq>${entry.priority}</changefreq>`,
+              entry.priority && `<priority>${entry.priority}</priority>`,
```

Comment:
Great point, thanks for noticing and fixing this! Can you add a test for this?

Reply:
done